docs/openapi/pipeline.yaml

```diff
@@ -5790,20 +5787,20 @@ components:
                     type: string
                     example: "ami-4d485ca7"
 
-        UpdateAKCSProperties:
+        UpdateACKProperties:
```

Comment:
Should be `UpdateACKProperties` I guess.

Reply:
#1612